src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -507,6 +509,24 @@ public async Task<IActionResult> TaskList(Guid applicationId)
             return View("~/Views/Roatp/TaskList.cshtml", model);
         }
 
+        private async Task<List<NotRequiredOverrideConfiguration>> PopulateNotRequiredOverridesWithApplicationData(Guid applicationId, List<NotRequiredOverrideConfiguration> notRequiredOverrides)
+        {
+            foreach (var overrideConfig in notRequiredOverrides)
+            {
+                var applicationDataValue = await _qnaApiClient.GetAnswerByTag(applicationId, overrideConfig.ConditionalCheckField);
+                if (applicationDataValue != null && applicationDataValue.Value != null)
+                {
+                    overrideConfig.Value = applicationDataValue.Value;
+                }
+                else
+                {
+                    overrideConfig.Value = string.Empty;
+                }
+            }
+
```

Comment:
if fact whole lot could be:
overrideConfigValue = applicationDataValue?.Value ?= null ? applicationDataValue : string.Empty;

Reply:
I prefer not to use the ternary operator, writing out long hand is more instantly readable